Cargo.toml

```diff
@@ -131,5 +129,10 @@ version = "0.1.3"
 default-features = false
 features = ["channel"]
 
+[workspace.dependencies.hyper-util]
+version = "0.1"
+default-features = false
+features = ["tokio", "tracing"]
```

Comment:
NB: https://github.com/hyperium/hyper-util/pull/166/files#diff-bdb64647cd31ba75babe48b3287899e6116a0a8c8267624ba227f86df5937467

to prevent any footguns wherein we might neglect to set this feature flag, and inadvertently fail to propagate trace context, i've set this at the workspace level.